main/lsp/requests/completion.cc

```diff
@@ -1209,6 +1209,13 @@ unique_ptr<ResponseMessage> CompletionTask::runRequest(LSPTypecheckerInterface &
             }
         }
 
+        ENFORCE(fref.exists());
+        auto level = fref.data(gs).strictLevel;
+        if (level < core::StrictLevel::True) {
+            items.emplace_back(getCompletionItemForUntyped(gs, queryLoc, 0, "(file is not typed)"));
```

Comment:
this leaves a little bit of ambiguity to people who are new to sorbet. we might want to say something like "(file is `# typed: false`)" or "(file is not `# typed: true` or higher)" to make it more clear.

Reply:
I was hoping that I wouldn't have to repeat the dance that hover needed to get the error message more precise, but I guess we should.

Unfortunately, I don't think we can use markdown for this, but maybe the markdown syntax is so ubiquitous people will be fine with it.

Reply:
yeah up to you

Reply:
Another thing you can consider is adding a documentation item so that when people click the (i) or press ^Space they'll see a longer explanation (that can use markdown, and also is not constrained to be a single short line)